docs/content/manpages/geowave-dbscan.txt

```diff
@@ -0,0 +1,144 @@
+//:geowave-dbscan(1)
+//:======================
+//::doctype: manpage
+
+NAME
+//:----
+
+geowave analytic dbscan - Density Based Scanner
+
+SYNOPSIS
+//:--------
```

Comment:
Small warning that it looks like Eric's pending pull request might change some of these arguments....